Cargo.toml

```diff
@@ -31,6 +31,14 @@ members = [
 default = []
 ion-hash = ["digest"]
 
+# Feature for indicating particularly bleeding edge APIs or functionality in the library.
+# These are not guaranteed any sort of API stability and may also have non-standard
+# Ion behavior (e.g., draft Ion 1.1 capabilities).
+experimental = ["experimental-streaming"]
+
+# Experimental streaming APIs
+experimental-streaming = []
```

Comment:
As per discussion with @popematt, added `experimental-streaming` to capture the experimental feature of the API that this supports  `experimental` now depends on `experimental-streaming`.